library/ix-dev/charts/elastic-search/Chart.yaml

```diff
@@ -15,12 +14,12 @@ maintainers:
 dependencies:
   - name: common
     repository: file://../../../common
-    version: 1.0.8
+    version: 1.0.9
 home: https://www.elastic.co
 icon: https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt280217a63b82a734/6202d3378b1f312528798412/elastic-logo.svg
 sources:
   - https://www.elastic.co/
-  - https://github.com/truenas/charts/tree/master/charts/elastic-search
+  - https://github.com/truenas/charts/tree/master/library/ix-dev/charts/elastic-search
```

Comment:
Why are we changing this path ?

Reply:
Because this will always show the readme.
If we leave the URL pointing to the catalog, in order to show the readme directly we will have to update the chart version in the link each time

Reply:
I wonder if we should publish latest version readme under `elastic-search` directory directly instead?
The path currently being used for development will be changed in the future as this is for library charts essentially..
What do you think about above proposal instead?

Reply:
Sounds good

Reply:
@Qubad786 Fixed some links the other way around in this case.